app/pages/Auction/BidActionPanel/OpenBid.js

```diff
@@ -31,7 +31,7 @@ class OpenBid extends Component {
 
     try {
       await sendOpen();
-      this.props.showSuccess('Successfully opened bid! Check back in a few minutes to start bidding.');
+      this.props.showSuccess('Successfully opened auction! Check back in a few minutes to start bidding.');
```

Comment:
Switched the phrasing here and in the portfolio view. Just a nit, it's not technically a bid yet

Reply:
👍